sensor_msgs/include/sensor_msgs/distortion_models.hpp

```diff
@@ -31,6 +31,9 @@
 // ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
 // POSSIBILITY OF SUCH DAMAGE.
 
+// This file is originally ported from ROS1:
```

Comment:
not sure "ported" applies here, there is nothing ROS (1 or 2) specific in this file